src/extension.ts

```diff
@@ -40,7 +40,7 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	// observer for logging workspace folder addition/removal
 	const logObserver = workspaceContext.observerFolders((folder, operation) => {
```

Comment:
```suggestion
	const logObserver = workspaceContext.observerFolders((folderContext, operation) => {
```

Maybe that would clear up the `folder.folder` references.